### PR TITLE
roll update sample typos

### PR DESCRIPTION
**Pull request by vikram-redhat**
@bcpswope this fixes the typo with roll update sample file reference and also adds it to the master.adoc as it was being referenced but not being included. Hope this makes sense.

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@bcpswope sorry, I included the removal of a duplicate header as well in this same PR.

File: security/create_nginx_policy.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ Complete the following steps to create a pod nginx policy from the CLI:
    kubectl get podnginxpolicy --namespace=<namespace>
    ```
 
-Your image pod nginx policy is created from the CLI. 
+Your image pod nginx policy is created from the CLI.
 
-#### Viewing your policy from the CLI 
+#### Viewing your nginx policy from the CLI
```
Comment (bcpswope) [inline security/create_nginx_policy.md:34]: this looks good @dockerymick FYI

File: manage_applications/deployment_rollout.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ The rolling update for a deployment runs on the hub cluster. The deployment roll
 You can configure a percentage rollout for a subscription or directly for a deployable resource.
 
   - Configuring a rolling update for a subscription
-  - Rolling update sample](roll_update_sample.md)
+  - [Rolling update sample](roll_update_sample.md)
```
Comment (bcpswope) [inline manage_applications/deployment_rollout.md:17]: I should have removed this--rolling update last minute removal for tech preview.

Comment (bcpswope) [reply manage_applications/deployment_rollout.md:16]: (I will have to catch up this content, there is a very long google doc with a bunch of changes but the release notes issues keep popping up-- this topic will change but for now, just removing that link.)

**Comment by bcpswope** [comment]
@vikram-redhat hoping these changes are ok--I needed to remove rolling update sample, please see what I changed.

File: security/create_nginx_policy.md (hunk 1 above)
Comment (vikram-redhat) [reply security/create_nginx_policy.md:34]: As a background, there was already a Viewing your policy from the CLI title somewhere else that was conflicting.

**Comment by bcpswope** [comment]
Feel free to merge if you can.

File: security/create_nginx_policy.md (hunk 1 above)
Comment (dockerymick) [reply security/create_nginx_policy.md:34]: @vikram-redhat thanks for catching this! @bcpswope thank you :)

**Comment by vikram-redhat** [comment]
I don't think I have merge rights :)

**Comment by bcpswope** [comment]
We will fix that! Lol.

**Pull request merged**